core-java/src/main/java/com/baeldung/utils/Utils.java

```diff
@@ -0,0 +1,34 @@
+package com.baeldung.utils;
+
+import java.util.Arrays;
+
+public class Utils {
```

Comment:
UtilsTest is a little bit too generic name. I can imagine multiple classes with similar names after having published hundreds of articles :)